src/main/java/tech/Astolfo/AstolfoCaffeine/main/cmd/fun/RandomAstolfo.java

```diff
@@ -0,0 +1,65 @@
+package tech.Astolfo.AstolfoCaffeine.main.cmd.fun;
+
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import com.jagrosh.jdautilities.command.Command;
+import com.jagrosh.jdautilities.command.CommandEvent;
+
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.Scanner;
+
+public class RandomAstolfo extends Command {
+    public RandomAstolfo() {
+        super.name = "astolfo";
+        super.aliases = new String[]{"astolfo"};
+        super.help = "sell shares on the market";
+        super.category = new Category("ancap");
+        super.arguments = "<stock> <amt>";
+    }
+
+    @Override
+    protected void execute(CommandEvent e) {
+
+        String inline = "";
+        try {
+            URL url = new URL("https://astolfo.rocks/api/v1/images/random");
+            HttpURLConnection conn = (HttpURLConnection)url.openConnection();
+            conn.setRequestMethod("GET");
+            conn.connect();
+            int response = conn.getResponseCode();
+            if(response != 200) {
+                throw new RuntimeException("HttpURLConnection " + response);
+            }else {
+                Scanner sc = new Scanner(url.openStream());
+
+                while(sc.hasNext()) {
+                    inline += sc.nextLine();
+                }
+                sc.close();
+
+                String image_link = "";
+                while(true){
+                    JsonElement jelement = new JsonParser().parse(inline);
```

Comment:
This is deprecated, please update your methods to the latest refactor.

Reply:
Unfortunately, of my knowledge, there aren't any good JSON parsing libraries in Java. The one I used is the best one that I know of

Reply:
Yes, I'm aware GSON is the best library but you're using methods the library maintainers have marked as deprecated, please use the appropriate methods that the library developers intend for you to use.

Reply:
smh, imagine using deprecated methods. If the GSON maintainers see us doing this, they'll never forgive us